config/vufind/ILS_common.ini

```diff
@@ -0,0 +1,8 @@
+; Common configurations usable for any ILS driver.
```

Comment:
I don't understand how this file is supposed to work; as far as I can tell, it's not loaded or used anywhere. Is this just for reference? If so, maybe it's better to just paste these lines into all of the ILS configs where they apply. I realize that's a lot of redundant work, but I think it's the best way to make the setting visible without making config loading more complicated. (It also makes sense for this to be in the driver-specific configs rather than config.ini since in a multi-ILS situation, there might be desire to have different debug settings for different drivers).

Reply:
Yea, that was my initial thought to avoid copy-pasting the same setting to each driver, but can do :)

Reply:
Sorry to make extra work, but I do think this will better ensure that people see the setting and know that it exists. :-)